yang/frr-bfdd.yang

```diff
@@ -233,6 +233,66 @@ module frr-bfdd {
     }
   }
 
+  grouping bfd-monitoring {
+    description
+      "BFD monitoring template for protocol integration.";
+
+    leaf enable {
```

Comment:
Why do we need this leaf? This grouping is always used in presence containers. We should either remove this leaf or make containers non-presence. The second option should be simpler because current create/destroy callbacks for presence containers don't actually do what they are supposed to do.